tests/blackbox/stratisd_cert.py

```diff
@@ -150,32 +150,32 @@ def test_pool_add_cache(self):
         """
         Test adding cache to a pool.
         """
-        cache_disks = DISKS[1:2]
+        cache_disk = DISKS[2:3]
 
         pool_name = p_n()
-        pool_path = make_test_pool(pool_name)
+        pool_path = make_test_pool(pool_name, DISKS[0:2])
 
-        (_, return_code, _) = StratisDbus.pool_add_cache(pool_path, cache_disks)
+        (_, return_code, _) = StratisDbus.pool_add_cache(pool_path, cache_disk)
```

Comment:
Just eliminate the ```cache_disk``` identifier and substitute it in the expression as:

```
(_, return_code, _) = StratisDbus.pool_add_cache(pool_path, DISKS[2:3])
```

That way you don't have to worry about your naming...if it's a list, even one of just one element, should the identifier really be singular or should it be plural?